ONSdigital/dp-frontend-release-calendar
# Adjust document type styling and spacing on Release page

### What

Adjust document type styling and spacing on Release page per Figma design.

<img width="793" alt="Screenshot 2022-05-26 at 17 47 33" src="https://user-images.githubusercontent.com/912770/170685725-31dce150-e9d0-4e9d-8ec8-4e31dd1d7e9c.png">

### How to review

- In a separate shell, run dp-design-system with `make debug`
- Run this frontend with `make debug`
- View a Release
- Verify that the colour and spacing matches the Figma design

### Who can review

Frontend / design system developers
